packages/server-boilerplate/src/orchestrator/api/ApiBuilder.ts

```diff
@@ -62,6 +63,17 @@ export class ApiBuilder {
     this.attachVerifyLogin = emptyMiddleware;
     this.verifyAuthMiddleware = emptyMiddleware; // Do nothing by default
     this.attachAccessPolicy = buildAttachAccessPolicy(new AccessPolicyBuilder(this.models));
+
+    // Dynamically set trusted proxy so that we can trust the IP address of the client
+    publicIp
+      .v4()
+      .then(publicIp => {
+        this.app.set('trust proxy', ['loopback', process.env.AWS_TRUSTED_PROXY_IP, publicIp]);
```

Comment:
```suggestion
    const trustedProxyIPs = process.env.TRUSTED_PROXY_IPS ? process.env.TRUSTED_PROXY_IPS.split(',') : [];
    
    // Dynamically set trusted proxy so that we can trust the IP address of the client
    publicIp
      .v4()
      .then(publicIp => {
        this.app.set('trust proxy', ['loopback', ...trustedProxyIPs, publicIp]);
```

Might suggest just this rename to keep things flexible. And nice to avoid explicitly referring to AWS infrastructure